pkgs/misc/vim-plugins/overrides.nix

```diff
@@ -395,6 +395,15 @@ self: super: {
     passthru.python3Dependencies = ps: with ps; [ beancount ];
   });
 
+  vim-closer = super.vim-closer.overrideAttrs(old: {
+    patches = [
+      (substituteAll {
+        src = ./patches/vim-closer/fix-tag.patch;
```

Comment:
Can we use a substituteInPlace or fetch an upstream patch?

Reply:
better yet, there's nothing nixos specific in this, please upstream this

Reply:
thanks for feedback, fixed